modules/proc_installing-the-rollout-manager-operator.adoc

```diff
@@ -0,0 +1,42 @@
+:_mod-docs-content-type: PROCEDURE
```

Comment:
As per the discussion during the call, this entire procedure has to be removed and user has to be instructed that the steps mentioned in the procedure at the following link: https://docs.openshift.com/gitops/1.11/argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.html#gitops-creating-rolloutmanager-custom-resource_using-argo-rollouts-for-progressive-deployment-delivery